shared/common-adapters/button.native.js

```diff
@@ -50,17 +50,15 @@ class Button extends Component<void, Props, void> {
 
     // Need this nested view to get around this RN issue: https://github.com/facebook/react-native/issues/1040
     return (
-      <NativeTouchableHighlight
```

Comment:
We have real inconsistency with activeOpacity so i'm just letting that not be plumbed through. i think in the past we were just making this up as it wasn't defined by design. if we need to change this lets do that later